idr0004-thorpe-rad52/idr0004-study.txt

```diff
@@ -16,7 +16,7 @@ Study Public Release Date	2016-04-27
 # Study Publication												
 Study PubMed ID	21893595	18085829										
 Study Publication Title	Bringing Rad52 foci into focus.	Genome-wide analysis of Rad52 foci reveals diverse mechanisms impacting recombination.										
-Study Author List	Thorpe PH, Alvaro D, Lisby M, Rothstein R.	Alvaro D,  Lisby M,  Rothstein R.										
+Study Author List	Thorpe PH, Alvaro D, Lisby M, Rothstein R	Alvaro D, Lisby M, Rothstein R										
```

Comment:
Why the duplication here?

Reply:
That's because 2 publications have been associated with the study hence two lists of authors. `idr0016` is another similar case.

Reply:
That's kinda confusing and looks like a bug. Since this is a list of authors for a *Study*, I wouldn't expect that a single author could be listed twice. If the Study were to link to a Publication then you could have "Thorpe PH, Alvaro D, Lisby M, Rothstein R" for one publication and "Alvaro D, Lisby M, Rothstein R" for the other. What benefit do you get by keeping 2 lists in the study (who uses that)?

Reply:
Thanks @will-moore , now I understand what you mean. Immediate thoughts:

- I am almost certain `Study Author List` is intended to be read as `Study Publication Author List` i.e. a tab-separated list of comma-separated authors associated with each publication.
- the 1-to-many study/publication mapping has already been mentioned as confusing - see https://github.com/IDR/idr-metadata/issues/375#issuecomment-503160347. I do not have enough background on the original rationale for this choice and I do not know anyone came up with a complete proposal to address the issue and upgrade published studies yet. 
- for the gallery UI specifically, one thought is that we could restrict the title/authors search to the first publication of each study, i.e. the principal one

This topic might be worth discussing at a weekly IDR weekly meeting and/or the issue linked above.